Dockerfile

```diff
@@ -1,33 +1,17 @@
-# Set the base image to Ubuntu
-FROM ubuntu
+FROM python:3.9-slim-buster
 
-# File Author / Maintainer
-MAINTAINER Couchbase
```

Comment:
should we retain the MAINTAINER here? I don't know what this is used for.

Reply:
`MAINTAINER` seems deprecated 🤷‍♀️ (my Vscode extension pointed out) , https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

I think I can replace with LABEL though 👍